### PR TITLE
Reddit is beta-testing 2FA

### PR DESCRIPTION
**Pull request by haykam821**
Reddit has started enabling a software-generated token. Because of this, I made this pull request but made the exceptions state it's only for beta testers.

image

### CODE REVIEW

File: _data/social.yml [hunk 1]
```diff
@@ -185,7 +185,10 @@ websites:
       url: https://www.reddit.com/
       twitter: reddit
```
Comment (mxxcon) [inline _data/social.yml]: Remote `twitter` if `tfa:yes`

**Review by mxxcon — DISMISSED** [review]

**Comment by jamcat22** [comment]
Additional info seems to be available here: https://www.reddit.com/r/modnews/comments/6x1uh7/twofactor_authentication_beta_for_moderators/

**Comment by stephengroat** [comment]
since this is for moderators only and the such a small subset of their users, i'm not sure this one would be ready to post.

**Comment by mxxcon** [comment]
If you join beta thing on reddit and go to their beta subreddit, you'll see some posts about tfa too..

**Comment by jamcat22** [comment]
Maybe we should use a status label?

**Comment by RichJeanes** [comment]
I think it's fair to add this since it is a pretty big deal for moderators, more so than your every day user. A user gets their account hacked, they might lose some fake internet points. A mod gets their account hacked and a whole community (potentially) gets nuked. But, yes, put an exception text on it that it is beta and mod-only at this time.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
As per [331440306](https://github.com/2factorauth/twofactorauth/pull/2739#issuecomment-331440306) and [331475143](https://github.com/2factorauth/twofactorauth/pull/2739#issuecomment-331475143), add a note that it's only applied to moderators or set tfa to no.

**Comment by Carlgo11** [comment]
Closing due to inactivity.

**Pull request closed without merging**